bind.go

```diff
@@ -30,6 +30,19 @@ type (
 // Bind implements the `Binder#Bind` function.
 func (b *DefaultBinder) Bind(i interface{}, c Context) (err error) {
 	req := c.Request()
+
+	paramNames := c.ParamNames()
+	paramValues := c.ParamValues()
+	params := make(map[string][]string)
+	for i, name := range paramNames {
+		// Fix for an echo bug where a param name would show up which dont exist if its name contains "id" e.g. "userid"
+		names := strings.Split(name, ",")
```

Comment:
```go
	// BindUnmarshaler is the interface used to wrap the UnmarshalParam method.
	BindUnmarshaler interface {
		// UnmarshalParam decodes and assigns a value from an form or query param.
		UnmarshalParam(param string) error
	}
```

and example:

```go
type StringArray []string

func (a *StringArray) UnmarshalParam(src string) error {
	*a = StringArray(strings.Split(src, ","))
	return nil
}
```

Reply:
Do you mean I should implement the bind param method via `UnmarshalParam`?
Or do you mean I should run the map of mapped params through `UnmarshalParam` to fix that bug I described?